modules/bookClass.js

```diff
@@ -0,0 +1,62 @@
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Date.now().toString();
+  }
+
+  static add() {
+    const title = document.querySelector('#title');
+    const author = document.querySelector('#author');
+    const addBtn = document.querySelector('#add-btn');
+    const errorMessage = document.querySelector('#message-alert');
+    const addMessage = document.querySelector('#add-message');
+
+    addBtn.addEventListener('click', (e) => {
+      e.preventDefault();
+      let awesomeBooks = [];
+      awesomeBooks = JSON.parse(localStorage.getItem('awesomeBooks') || '[]');
+      if (title.value !== '' && author.value !== '') {
+        const newBook = new Book(title.value, author.value);
+        awesomeBooks.push(newBook);
+        localStorage.setItem('awesomeBooks', JSON.stringify(awesomeBooks));
+        // window.location.reload();
```

Comment:
[OPTIONAL] Consider removing this commented line to keep your codebase even more organized.

Reply:
I removed the comment now. Thank you.